packages/shared/charts/src/ui/Playground.scss

```diff
@@ -0,0 +1,88 @@
+.widget-charts-playground {
+    padding: 5px;
+
+    .ace_editor {
+        margin-bottom: 0;
+        width: 100% !important;
```

Comment:
In that case, maybe it's better to change the default value in the `ace configuration`?
[Available Props](https://github.com/securingsincity/react-ace/blob/master/docs/Ace.md#available-props)
It is shown on `px` but I believe using `%` is also a option, because it accepts string

Reply:
I'll take a look